tests/FIP-48-BD-4650-reallocate-tokens-tests.js

```diff
@@ -0,0 +1,595 @@
+    publicKey: 'FIO7DVJTbCjmBev9oAggRP7kRjXJfZAE4228SnMfsyyBhv7wggxDB',
+    fioAmount: 10000000000000000
+  }
+  const account3Info = {
+    account: 'hjvwdy5p4zvs',
+    publicKey: 'FIO7BQfxKtfebgrTZYbMYi6gvcWZwU8dogL78jAwNgeQVn5xazKAv',
+    fioAmount: 7000000000000000
+  }
+  const account4Info = {
+    account: '2mskjvkhj334',
+    publicKey: 'FIO5PyTGT7AitAcCEcmVhwfsWQ6DnGhai73wgGA4wxk7KQfpck9YX',
+    fioAmount: 5500000000000000
+  }
+  const account5Info = {
+    account: 'oadme4v54cly',
+    publicKey: 'FIO6z6cPxUx9S7r9qaPzLZCsWADYha8RbXcL58vcGfLg89JdssBv3',
+    fioAmount: 2500000000000000
+  }
+  const account6Info = {
+    account: 'jsniuyaaeblr',
+    publicKey: 'FIO5FZVQ6ehswr1Mig5a18r3FwdfFmXDVjn6TrAKLHZa3KUrpvSs3',
+    fioAmount: 1999999400000000
+  }
+  const account7Info = {
+    account: 'nadppzyxtxjx',
+    publicKey: 'FIO5HBTJeDZqbmVNXJy9CxJyKyeg6aLXMtfXoi7GddftjnFf3UKUt',
+    fioAmount: 1500000000000000
+  }
+  const account8Info = {
+    account: 'zvt11xu5czlk',
+    publicKey: 'FIO7K8wwggViLpSoEKXUbLirCpr7eFprWnaxq8ZMG7remasvmuGYo',
+    fioAmount: 1000000000000
+  }
+  const account9Info = {
+    account: 'dioxleem5hmr',
+    publicKey: 'FIO6R6JTP1CNsX4PyNwPXf3RWQ4NswRV1P4wpVtn8Fu5hqdbmjaAu',
+    fioAmount: 1000000000000
+  }
+  const account10Info = {
+    account: 'iud1tjwtt2ey',
+    publicKey: 'FIO7v6KMR6imo4EqZUBEJA51xL4ZjEaZtxNBT7zbHPm45mNnQGvBP',
+    fioAmount: 1000000000000
+  }
+  const account11Info = {
+    account: 'xgyg22tfizja',
+    publicKey: 'FIO82nTSBe3CHeZnAACqHwLFSUENZGJd7h6Fo5rAzV6E2mvadFWzB',
+    fioAmount: 1000000000000
+  }
+  const account12Info = {
+    account: '4urqjmtfvmjj',
+    publicKey: 'FIO5QebHVoKoETsA9aGXET5XdAeGwA1AcqRHGxawgGfDNPZwuwdtJ',
+    fioAmount: 1000000000000
+  }
+  const account13Info = {
+    account: 'deq54dxuyquh',
+    publicKey: 'FIO8MmqVkbBEj6vkXFuQRM8p6AC94zJCWsLHxmUengkUvSwD7Fths',
+    fioAmount: 1000000000000
+  }
+  const receiveraccountInfo = {
+    account: 'pkfbwyi2qzii',
+    publicKey: 'FIO6WYaLQzB196NAyrfqCBQzJXYkDk99B2by1F8MHyEbAWHYJRnK2',
+    fioamount: 1000000000
+  }
+
+
+
+
+
+
+  it(`Create account`, async () => {
+    try {
+      userA1 = await newUser(faucet);
+    }catch (err){
+      console.log("ERROR CREATING USERS " + err);
+    }
+  });
+
+
+  //account 1 start
+  it(`Transfer  tokens to account1`, async () => {
+    const result = await faucet.genericAction('transferTokens', {
+      payeeFioPublicKey: account1Info.publicKey,
+      amount: account1Info.fioAmount,
+      maxFee: config.api.transfer_tokens_pub_key.fee,
+      technologyProviderId: ''
+    })
+    expect(result.status).to.equal('OK')
+  });
+
+  it(`Lock  tokens account1 lock type 2`, async () => {
+
+    const result1 = await userA1.sdk.genericAction('pushTransaction', {
+      action: 'addlocked1',
+      account: 'eosio',
+      data: {
+        owner: account1Info.account,
+        unlockperiodcount: 0,
+        amount: account1Info.fioAmount,
```

Comment:
There are a few accounts that have a different total_grant_amount than remaining_locked_amount, because they were able to spend it on fees. Should we actually set total_grant_amount and remaining_locked_amount for each account separately to match Mainnet?

Similarly:
1. Some accounts voted (are in voters table). Should we match?
2. Some accounts executed some transactions and paid fees? Should we match?
3. Some accounts receives tokens above their locked amount? Should we match?